packages/lesmis/src/views/AdminPanel/SurveyResponsesView.js

```diff
@@ -49,37 +55,19 @@ const COLUMNS = [
   {
     Header: 'Reject',
     source: 'id',
+    Cell: RejectButton,
     type: 'delete',
     actionConfig: {
       endpoint: 'surveyResponses',
     },
   },
 ];
 
-// eslint-disable-next-line react/prop-types
-const ConfirmRejectModal = ({ isOpen, onConfirm, onCancel }) => {
-  return (
-    <ConfirmModal
-      onClose={onCancel}
-      isOpen={isOpen}
-      handleAction={onConfirm}
-      isLoading={false}
-      title="Reject Survey Response"
-      mainText="Are you sure you want to reject this Survey Response?"
-      description="Rejecting a Survey Response will also delete it. Once deleted this can’t be undone."
-      actionText="Yes, Reject and Delete"
-      loadingText="Saving"
-    />
-  );
-};
-
-// Todo: Replace base filter with real filter @see WAI-832
 export const DraftSurveyResponsesView = props => (
   <SurveyResponsesPage
     {...props}
     title="Survey Responses For Review"
-    baseFilter={{ 'survey.code': { comparator: 'NOT ILIKE', comparisonValue: '%_Confirmed_WNR' } }}
+    baseFilter={{ approval_status: { comparisonValue: 'pending' } }}
```

Comment:
We probably do need a fourth page to show those that don't go through an approval workflow at all

Reply:
hmmm maybe the not required ones should just go into the approved tab?

Reply:
Good thought! I just chatted to @enunan and she prefers having a separate table for those that don't require approval